hibernate-envers/src/test/java/org/hibernate/envers/test/integration/strategy/ValidityAuditStrategyRevEndTsTest.java

```diff
@@ -476,7 +476,7 @@ private void verifyRevEndTimeStamps(String debugInfo, List<Map<String, Object>>
 			}
 			else {
 				if ( getDialect() instanceof MySQL5Dialect &&
-						!( getDialect() instanceof MySQL57InnoDBDialect || getDialect() instanceof MariaDB53Dialect) ) {
+						!( getDialect() instanceof MySQL5Dialect || getDialect() instanceof MariaDB53Dialect) ) {
```

Comment:
This change here looks like it could potentially break this test.  

We previously had {{MySQL5InnoDBDialect}} and {{MySQL57InnoDBDialect}} and the comments dictate that MySQL5 behaves differently than MySQL 5.7.  I'm not sure whether someone has tested this change on various versions of MySQL to be sure but this change seems to make MySQL5 and MySQL 5.7 behave the same now.